aea/agent.py

```diff
@@ -164,22 +179,13 @@ def timeout(self) -> float:
     @property
     def loop_mode(self) -> str:
         """Get the agent loop mode."""
-        return self._loop_mode
-
-    @property
-    def main_loop(self) -> BaseAgentLoop:
-        """Get the main agent loop."""
-        return self._main_loop
+        return self._loop_mode or self.runtime.DEFAULT_RUN_LOOP
```

Comment:
Do we need this property? It looks like the info should be passed straight to the runtime constructor and be available on the runtime as a property!

Reply:
moved to runtime.
done